pkg/authentication/firebase.go

```diff
@@ -0,0 +1,46 @@
+package authentication
+
+import (
+	"context"
+	"firebase.google.com/go/v4/auth"
+)
+
+// FirebaseTokenVerifier contains a method that allows verifying a Token signed by Firebase.
```

Comment:
For future reference, whenever you document an interface, document its purpose and maybe some use case examples instead of what its methods do. The names of the methods and their documentation should explain what they do.

```suggestion
// FirebaseTokenVerifier verifies a Token signed by Firebase.
```

Reply:
The interface to created in order to mock the SDK when testing, no other than that. I didn't think it was relevant to add that as documentation.

Reply:
I actually find that quite helpful. I came to that conclusion after re-reading the code after sending this comment.

Still, the main comment stands: don't comment on what methods the interface contains, but mention what it does at a high level and any usage comments.

Reply:
Care to give this another pass?

Reply:
Did you choose to not address this?

Reply:
My bad, I addressed this comment in a different PR. I've just cherry-picked the changes from it. Sorry for that.